src/executors/app-build-suite.yaml

```diff
@@ -0,0 +1,3 @@
+docker:
+  - entrypoint: /bin/bash
+    image: quay.io/giantswarm/app-build-suite:0.0.0-53eb22a7e5ce2f558b1e57b5eb6a10429f57837a-circleci
```

Comment:
I know it's still in dev but is the plan to have tagged releases for ABS?

Reply:
yes, once

- we release `0.18.0` of architect-orb -> I am doing this at this moment
- we resolve some issues with automation coming from `giantswarm/github`

Reply:
OK cool 👍